clang/test/CXX/drs/cwg22xx.cpp

```diff
@@ -108,6 +108,7 @@ namespace MultilevelSpecialization {
   template<> template<int a, int b>
     void B<int, int>::f(int i, int (&arr1)[a], int (&arr2)[b]) {}
     // since-cxx11-error@-1 {{out-of-line definition of 'f' does not match any declaration in 'cwg2233::MultilevelSpecialization::B<int, int>'}}
+    // expected-note@-6 {{defined here}}
```

Comment:
```suggestion
    //   expected-note@#cwg2233-blah {{defined here}}
```
We shouldn't make the readers count lines to understand, which line the diagnostic is emitted for. Use bookmarks instead (with a proper name, see other tests for the reference).

Reply:
Yeah, adding a diagnostic that’s very general like this one tends to require updating a lot of tests.